plugins/jaeger/wiring.go

```diff
@@ -13,10 +21,14 @@ import (
 	"github.com/blueprint-uservices/blueprint/blueprint/pkg/wiring"
 )
 
-// Generates the IRNodes for a jaeger docker container named `collectorName` that uses the latest jaeger:all-in-one container
-// and the clients needed by the generated application to communicate with the server.
+// Adds a jaeger docker container named `collectorName` that uses the latest jaeger:all-in-one container
```

Comment:
I think it's important to preface wiring spec functions with the following:

>  [Function] can be used by wiring specs to ...

in this case, 

> [Collector] can be used by wiring specs to instantiate a Jaeger trace collector with name `collectorName`.  The collector is a container-level instance that uses the latest jaeger:all-in-one image.